src/tests/ftest/vmd/led.py

```diff
@@ -107,10 +107,11 @@ def test_vmd_led_faulty(self):
             uuid_list = sorted(uuid_dict.keys())
             self.log.info("Devices on hosts %s: %s", hosts, uuid_list)
             for uuid in uuid_list:
-                resp = set_device_faulty(self, self.dmg, hosts.split(':')[0], uuid)
+                _sys_xs = uuid_dict[uuid]['has_sys_xs']
+                set_device_faulty(self, self.dmg, hosts.split(':')[0], uuid, None, _sys_xs)
                 self.log.info("Sleeping for 15 seconds ...")
                 time.sleep(15)
-                self.log.info(resp)
+                self.log.info(self.dmg.result)
```

Comment:
I disagree with this. Calling a function and expecting it to store the result somewhere else is bad practice IMO unless necessary for some other reason. It obfuscates the code, makes it easier to introduce subtle bugs, harder to debug, harder to refactor later.

One example in this case is how can we even be sure that the response printed  in loop iteration 1+ is actually from that loop and not from loop 0? I think it's better to not introduce that complexity